lite-rpc/src/bridge.rs

```diff
@@ -334,4 +335,20 @@ impl LiteRpcServer for LiteBridge {
             Ok(None)
         }
     }
+
+    async fn get_epoch_info(&self) -> crate::rpc::Result<EpochInfo> {
```

Comment:
Does not look like the same definition in existing RPC. Should be somewhat similar, I see that there is missing commitment and minContext Slot.

https://docs.solana.com/api/http#getepochinfo

Reply:
I take the solana_rpc_client signature: https://docs.rs/solana-client/latest/solana_client/nonblocking/rpc_client/struct.RpcClient.html#method.get_epoch_info
I see that there's a [get_epoch_info_with_commitment](https://docs.rs/solana-client/latest/solana_client/nonblocking/rpc_client/struct.RpcClient.html#method.get_epoch_info_with_commitment).
I use the solana client API or the RPC doc?